### PR TITLE
added button component

### PR DESCRIPTION
**Pull request by DilipGelot**
image

### CODE REVIEW

**Comment by DilipGelot** [comment]
image

image

File: src/app/components/Button.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+const Button = ({ type }) => {
+  const getButtonStyle = (type) => {
+    switch (type) {
```
Comment (DashrathGelot) [inline src/app/components/Button.jsx:5]: Change this switch, if it’s too many types then use json and get value based on key

File: src/app/globals.css [hunk 2]
```diff
@@ -1,33 +1,3 @@
...
-      rgb(var(--background-end-rgb))
-    )
-    rgb(var(--background-start-rgb));
-}
-
-@layer utilities {
-  .text-balance {
-    text-wrap: balance;
-  }
-}
```
Comment (DashrathGelot) [inline src/app/globals.css:33]: Why all these are being removed?

File: src/app/page.js [hunk 3]
```diff
@@ -1,113 +1,15 @@
...
-          </p>
-        </a>
-      </div>
+    <main className="container">
+      <Button type={"primary"} />
+      <Button type={"secondary"} />
+      <Button type={"third"} />
+      <Button type={"add_cart"} />
+      <Button type={"checkout"} />
+      <Button type={"back-top"} />
```
Comment (DashrathGelot) [inline src/app/page.js:12]: Don’t put all these this kind of testing

File: src/app/globals.css (hunk 2 above)
Comment (DilipGelot) [reply src/app/globals.css:33]: this css is used in NextJS welcome/default page

File: src/app/components/Button.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        return "bg-fourth text-primary";
+      case "checkout":
+        return "bg-fifth text-primary";
+      case "back-top":
+        return "bg-fifth border border-primary text-primary";
+      default:
+        return "bg-primary border border-red-600";
+    }
+  };
+  const btn_classes = getButtonStyle(type);
```
Comment (DashrathGelot) [inline src/app/components/Button.jsx:22]: use direct in tag

File: src/app/page.js (hunk 3 above)
Comment (DashrathGelot) [reply src/app/page.js:12]: remove this

**Pull request merged**